### ISSUE TITLE
Repeated Lag when using Bluetooth Headphones and Seeking Media

### ISSUE THREAD

**Issue by srvthegeek** [mpv-player/mpv#9243]
### Important Information

Provide following Information:
- mpv version: 0.33.1
- macOS Version: 11.6 (Big Sur)
- Source of the mpv binary or bundle: https://laboratory.stolendata.net/~djinn/mpv_osx/

### Reproduction steps

- Connect a Bluetooth Headset (Sony WH1000-XM3 to be precise).
- Open a video file that contains audio.
- Try seeking forward/backward (Starts to lag).
- Now open the same file with QuickTime/VLC.
- Let the file load and play for a bit (you may seek forward.backward too, it works smoothly without any lag).
- Pause the Quicktime/VLC/Chrome file
- Return to mpv and try seeking forward/backward now (It works smoothly without any lag).
- Quit QuickTime/VLC/Chrome.
- Return to mpv and try seeking forward/backward again (Starts to lag).

### Expected behavior
Seeking forward/backward in a video is smooth after initial loading when bluetooth headphones are connected. 

### Actual behavior
After a few seconds of opening a video file with bluetooth headphones connected, it seemingly starts to play fine, but whenever I seek to a new position or pause+play, it takes a few seconds to load that position. Additionally, this is mitigated if I load a file in QuickTime/VLC/Chrome in parallel (but starts to lag again when I quit Quicktime/VLC).
(This problem is not present on Quicktime/VLC/Chrome, they take a few seconds to load initially but seeking after that is lag-free)

### Log file
[https://0x0.st/-Y_m.log](https://0x0.st/-Y_m.log)

To get this log, I initially played the file without any other program running, tried seeking a few times (started to lag). I then switched to chrome and played the same file and paused, then returned to mpv to seek again without any lag.

### Sample files

File used while logging (Public Domain Video): [https://0x0.st/-Y_A.mp4](https://0x0.st/-Y_A.mp4)

**Comment by Akemi** [comment]
duplicate of #4141.

**Comment by forthrin** [comment]
@Akemi said this should be reopened. Reposting observation from #4141.

My observation with AirPods Pro 2 (Bluetooth):

1. Press → or ↑ to jump ahead in video.
2. Video jumps ahead instantly, but does not play immediately.
3. After 1 second or so, the video plays, and the audio is in sync.

Do note:

1. The error does not happen consistently. I see no pattern to when it happens.
2. If disabling video, the delay still happens with the audio.
3. If disabling audio, the delay is near-zero.

Note also:

1. With QuickTime Player (with Bluetooth), the delay is near-zero.
2. With wired speakers (without Bluetooth), the delay is near-zero.

The 1 second delay becomes very annoying when pressing the arrow keys multiple times to skip ahead to something.

**Comment by LampPrinter** [comment]
> @Akemi said this should be reopened. Reposting observation from #4141.
> 
> My observation with AirPods Pro 2 (Bluetooth):
> 
> 1. Press → or ↑ to jump ahead in video.
> 2. Video jumps ahead instantly, but does not play immediately.
> 3. After 1 second or so, the video plays, and the audio is in sync.
> 
> Do note:
> 
> 1. The error does not happen consistently. I see no pattern to when it happens.
> 2. If disabling video, the delay still happens with the audio.
> 3. If disabling audio, the delay is near-zero.
> 
> Note also:
> 
> 1. With QuickTime Player (with Bluetooth), the delay is near-zero.
> 2. With wired speakers (without Bluetooth), the delay is near-zero.
> 
> The 1 second delay becomes very annoying when pressing the arrow keys multiple times to skip ahead to something.
> 
> PS! I thought the reason for this was that there is latency in Bluetooth audio, and thus when jumping to a new position in the video, playback has to wait the same amount of milliseconds for video and audio to be in sync.
> 
> Wouldn't this be solved by starting video playback immediately, and give the audio a negative delay? (Provided mpv knows how long the real-life audio delay actually is. Is it approximately the same for all Bluetooth-devices?)


I have FiiL T1 Pro bluetooth headsets and there is a general 0.5 second delay when resuming a paused video. When on speakers, resume is instantaneous.
There does not seem to be any lag when seeking.

This Pull Request by @LiJChang fixes that.
https://github.com/mpv-player/mpv/pull/10953/commits/97810aabe1a428893dc6d5d7716452f563ecf8dc

**Comment by forthrin** [comment]
@LampPrinter: Right off the bat this seems to work. I could skip through videos with no delay whatsoever. Since the bug seemed intermittent, I'll post back if it does reoccur. Otherwise this seems suited for the main branch. Great work!
